tests/acceptance/users_frontend.feature

```diff
@@ -0,0 +1,46 @@
+Feature: Users Frontend
+  In order to manage users account in the web
+  As a user
+  I need to check user login and registration in joomla! CMS
+
+  Background:
+    Given I see the joomla! Home page
```

Comment:
> background scenario is not necessary here because our users.feature last scenario display joomla home page.

Interdependancies of tests should be avoided. If dependancies are needed, they MUST be made explicit (like the `@depends` annotation with PHPUnit).

I don't agree with Yves here. Being on the frontend start page (more precise than 'Joomla! Home Page', which I associate with joomla.org) is a valid and necessary pre-condition.